inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/HiveLoadNode.java

```diff
@@ -88,7 +90,7 @@ public HiveLoadNode(@JsonProperty("id") String id,
             @JsonProperty("hiveVersion") String hiveVersion,
             @JsonProperty("hadoopConfDir") String hadoopConfDir,
             @JsonProperty("parFields") List<FieldInfo> partitionFields) {
-        super(id, name, fields, fieldRelationShips, filters, sinkParallelism, properties);
+        super(id, name, fields, fieldRelationShips, filters, filterStrategy, sinkParallelism, properties);
```

Comment:
recommand every load node,every extract node  have a builder mode, so every time we add filed in 'LoadNode' or 'ExtractNode',We can avoid modifying the call site of the constructor call.
Better open an issue to follow it.